app/config.py

```diff
@@ -44,5 +55,20 @@ class Settings(BaseSettings):
     environment: str = "dev"
     migration_dir: Path = PROJECT_DIR / "migrations"
 
+    # OpenAPI server configuration
+    @property
+    def api_servers(self) -> list[dict]:
+        return [
```

Comment:
You can hardcode the staging and the production server (respectively https://nutripatrol.openfoodfacts.net and https://nutripatrol.openfoodfacts.org), and the associated descriptions.